test/examples/ucp_hello_world.c

```diff
@@ -51,10 +56,15 @@ struct ucx_context {
     int             completed;
 };
 
+enum ucp_test_mode_t {
+    TEST_MODE_PROBE,
+    TEST_MODE_POLL,
```

Comment:
This should be renamed to 'TEST_MODE_WAIT' or similar, since POLL may imply "busy possling"